regulations/static/regulations/css/scss/partials/_navigation.scss

```diff
@@ -37,13 +37,12 @@ Appendix Navigation
 
 .appendix-nav a:link,
 .appendix-nav a:visited {
-  color: #101820;
+  color: $black;
```

Comment:
Yep, $black is the main text color everywhere in the app and this was just a stray hex code.